data/emitter

```diff
@@ -1,9 +0,0 @@
-{"t":1608079158474,"m":"Screwdriver Launcher information","s":"sd-setup-launcher"}
```

Comment:
launch_test looks like to use this file. 
Can you delete this file?
https://github.com/screwdriver-cd/launcher/blob/d99488f7fe3723ffb4d29d345bf4de413b9bbbf0/launch_test.go#L27

Reply:
`./data/emitter` is created automatically, so we don't need it.
It looks like it was [deleted](https://github.com/screwdriver-cd/launcher/commit/24c4e3f4d7787113d831c096b88ac7e22358e08a#diff-a680012bcc3aec2d5edf07c5b0b31bbccef0bc5384d733ba73af2e54c9909dea) in the past, but it was restored. It's also listed in [.gitignore](https://github.com/screwdriver-cd/launcher/blob/d99488f7fe3723ffb4d29d345bf4de413b9bbbf0/.gitignore#L5).